src/taskpane/components/MergeComponents/MergeConflict.js

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import './MergeConflict.css';
+import { runSelectCell }  from "../../../saga/sagaUtils.js";
+ 
+
+/* global */
+
+// select the given cell in the Excel UI
+async function selectCell(e, sheet, cell) {
+    e.preventDefault()
+    runSelectCell(sheet, cell)
+}
+
+export default class MergeConflict extends React.Component {
```

Comment:
Let's make this a pure functional component. There are a few examples throughout the codebase, but in general it'll let us cut out a whole bunch of code and speed this up!